docs/integrations/sources/postgres.md

```diff
@@ -1,263 +1,273 @@
+- Log-based replication only works for master instances of Postgres.
+- Using logical replication increases disk space used on the database server. The additional data is stored until it is consumed.
+  - Set frequent syncs for CDC to ensure that the data doesn't fill up your disk space.
+  - If you stop syncing a CDC-configured Postgres instance with Airbyte, delete the replication slot. Otherwise, it may fill up your disk space.
 
-Follow one of these guides to enable logical replication:
+### Setting up CDC for Postgres​
 
-* [Bare Metal, VMs \(EC2/GCE/etc\), Docker, etc.](#cdc-on-bare-metal-vms-ec2-gce-etc-docker-etc)
-* [AWS Postgres RDS or Aurora](#cdc-on-aws-postgres-rds-or-aurora)
-* [Azure Database for Postgres](#cdc-on-azure-database-for-postgres)
+Airbyte requires a replication slot configured only for its use. Only one source should be configured that uses this replication slot. See Setting up CDC for Postgres for instructions.
 
-#### 2. Add user-level permissions
+#### Step 1: Enable logical replication​
 
-We recommend using a user specifically for Airbyte's replication so you can minimize access. This Airbyte user for your instance needs to be granted `REPLICATION` and `LOGIN` permissions. You can create a role with `CREATE ROLE <name> REPLICATION LOGIN;` and grant that role to the user. You still need to make sure the user can connect to the database, use the schema, and to use `SELECT` on tables \(the same are required for non-CDC incremental syncs and all full refreshes\).
+To enable logical replication on bare metal, VMs (EC2/GCE/etc), or Docker, configure the following parameters in the [postgresql.conf file](https://www.postgresql.org/docs/current/config-setting.html) for your Postgres database:
 
-#### <a name="replication-plugin"></a>3. Select replication plugin
+| Parameter  | Description  | Set value to |
+|------------|--------------|--------------|
+| wal_level  | Type of coding used within the Postgres write-ahead log  | logical |
+| max_wel_senders | The maximum number of processes used for handling WAL changes | Min: 1 |
+| max_replication_slots | The maximum number of replication slots that are allowed to stream WAL changes | 1 (if Airbyte is the only service reading subscribing to WAL changes. More than 1 if other services are also reading from the WAL) |
 
-We recommend using a `pgoutput` plugin as it is the standard logical decoding plugin in Postgres. In case the replication table contains a lot of big JSON blobs and table size exceeds 1 GB, we recommend using a `wal2json` instead. Please note that `wal2json` may require additional installation for Bare Metal, VMs \(EC2/GCE/etc\), Docker, etc. For more information read [wal2json documentation](https://github.com/eulerto/wal2json).
+To enable logical replication on AWS Postgres RDS or Aurora​:
 
-#### <a name="replication-slot"></a>4. Create replication slot
```

Comment:
@Amruta-Ranade, the internal anchors were deleted here. They are actually referenced in the `spec.json` in some of the field descriptions. I am adding them back in https://github.com/airbytehq/airbyte/pull/15382.